tests/functional-tests/utils/crd_properties.json

```diff
@@ -0,0 +1,207 @@
+{
```

Comment:
How is this file generated?

Reply:
@mew2057 , for now this file content is static and content has been generated by converting "deploy/crds/csiscaleoperators.csi.ibm.com.crd.yaml" file into json. As it was becoming difficult to add this code in python API so we are directly reading this json file to create crd api spec.

Reply:
We really should be using the operator's CRD, since otherwise these tests might not "notice" a change done by the operator.  (with that said, we have the molecule tests already)

Reply:
Sure @dunnevan, I should be able to make this change in next commit.